gpt_index/readers/file.py

```diff
@@ -31,7 +31,7 @@ def load_data(self, **load_kwargs: Any) -> List[Document]:
         data = ""
         data_list = []
         for input_file in self.input_files:
-            with open(input_file, "r") as f:
+            with open(input_file, "r", errors='replace') as f:
```

Comment:
I was poking around the docs `errors` and it seems error characters get replaced with a "?" (correct me if i'm wrong)

Have we considered `errors='ignore'` instead?